tests/test_tarball_generation.py

```diff
@@ -68,7 +68,7 @@
 
 # 5. Run and verify CLI
 try:
-    cli_cmd = [os.path.join(bin_dir, "uc"), "catalog", "create", "--name", "Test Catalog"]
+    cli_cmd = [os.path.join(bin_dir, "uc"), "catalog", "create", "--name", "Test_Catalog"]
```

Comment:
I don't think we should change this test case. Are commands args with spaces not working?

Reply:
UC does not allow objects with spaces in them, so the test with the space in the name _should_ be failing, and now is failing with @abrassel 's double quote changes, which is why I asked him to change this test. I don't know why this test is currently successful before this PR but I believe it should be failing.